HunterPie.Core/Game/Data/Schemas/AbnormalitySchema.cs

```diff
@@ -1,11 +1,19 @@
 ﻿namespace HunterPie.Core.Game.Data.Schemas;
 
+public enum CompareType
+{
+    WithValue,
+    WithValueNot
+}
+
 public struct AbnormalitySchema
 {
     public string Id;
     public int Offset;
     public int DependsOn;
+    public int CompareOpr;
```

Comment:
Why `int`? It's an enum. Also try not to abbreviate field names, that's against clean code practices.